modules/nw-secondary-ext-gw-object.adoc

```diff
@@ -0,0 +1,173 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/configuring-secondary-external-gateway.adoc
+
+:_content-type: CONCEPT
+[id="nw-secondary-ext-gw-object_{context}"]
+= AdminPolicyBasedExternalRoute object configuration
+
+You can define an `AdminPolicyBasedExternalRoute` object with the following properties.
+
+.`AdminPolicyBasedExternalRoute` object
+[cols=".^3,.^2,.^5a",options="header"]
+
+|====
+|Field|Type|Description
+
+|`from`
+|`string`
+|
+Specifies a namespace selector that the routing polices apply to. Only `namespaceSelector` is supported for external traffic. For example:
+
+[source,yaml]
+----
+from:
+  namespaceSelector:
+    matchLabels:
+      kubernetes.io/metadata.name: novxlan-externalgw-ecmp-4059
+----
+
+|`nextHops`
+|`object`
+|
+Specifies the destinations where the packets are forwarded to. Must be either or both of `static` and `dynamic`.
```

Comment:
Do you think it's worth specifyin there is a minimum of 1 hop required?

Reply:
@jordigilh, in the next hops static or dynamic array?

Reply:
There needs to be at least one, either static or dynamic. The next hops array cannot be empty.